source/plugin/configuration/nodes.rst

```diff
@@ -130,23 +139,25 @@ method with a ``TypeToken`` and the object you want to serialize.
     Serializing a value will throw an ``ObjectMappingException`` if no ``TypeSerializer`` for the given ``TypeToken``
     can be found.
 
-For simple classes like ``UUID``, you can just create a ``TypeToken`` using the static ``TypeToken.of()`` method.
-But when the class you want to use has type parameters of its own (like ``Map<String,UUID>``) the syntax gets a
+For simple classes like ``UUID``, you can just create a ``TypeToken`` using the static :javadoc:`TypeToken#of(Class)`
```

Comment:
An UUID TypeToken is provided by `TypeTokens` though.

Reply:
Added extra comment regarding that variable.